libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -79,6 +84,7 @@ + (instancetype)sharedManager
     static SalesforceSDKManager *sdkManager = nil;
     dispatch_once(&pred , ^{
         uid = [[[UIDevice currentDevice] identifierForVendor] UUIDString];
+        features = [NSMutableSet setWithObjects:kSFAppFeatureOAuth, nil];
```

Comment:
Should you set the oauth feature from the oauth code - I guess it's possible to do anonymous (unauthenticated) calls to the server.